src/main/scala/io/iohk/ethereum/jsonrpc/DebugJsonMethodsImplicits.scala

```diff
@@ -12,4 +19,19 @@ object DebugJsonMethodsImplicits extends JsonMethodsImplicits {
       def encodeJson(t: ListPeersInfoResponse): JValue =
         JArray(t.peers.map(a => JString(a.toString)))
     }
+
+  implicit val debug_getSpecificBlockRequest: JsonMethodCodec[GetSpecificBlockRequest, GetSpecificBlockResponse] =
```

Comment:
I think that this endpoint shouldn't be part of this PR.
1) it's not documented or tested
2) it's not part of the bugfix
3) will make it harder to backport to a release branch if the need arises

I'm fine with having this come from a ticket that's fleshed out and has its parameters properly identified, perhaps during a refinement session.

Reply:
Fair enough. I think this endpoint might be useful for state sync improvements, and we'll be able to grab it from this PR when that's the case.